src/lTerm_widget_callbacks.ml

```diff
@@ -44,7 +45,8 @@ let exec_callbacks seq x =
        try
          f x
        with exn ->
-         ignore (Lwt_log.error ~section ~exn "callback failed with"))
+         Log.err (fun m -> m "callback failed with %s" (Printexc.to_string exn));
```

Comment:
note that these two lines aren’t equivalent. One waits for the action to finish and the other doesn’t. I don’t know if it matter in this particular context but it’s maybe to take into account

Reply:
Right, the reason I didn't use Logs_lwt here is that it would change the signature of the function (which is exposed publicly). It would also force us to return an `Lwt.t` just for the logs.

I don't think it would matter here: worst case we'd see race conditions and the logs wouldn't be in the right order, but as this is just logging an exception in a widget callback, I don't think that's a problem.